### DISCUSSION TITLE
RSpec/AggregateExamples: customize example group alias

### DISCUSSION THREAD

**Discussion by tetienne** [test-prof/test-prof#250] [Ideas]
Currently, when this cop auto-correct is applied, the example group keyword is replaced by `specify`. It would be nice if the cop is smart enough to detect the one used by the test, or at least if the alias can be setup in the cop configuration.

Currently
```ruby
specify(:aggregate_failures) do
        expect(instance).to respond_to(:copy)
end
```

Wished

```ruby
it(:aggregate_failures) do
        expect(instance).to respond_to(:copy)
end
```

**Comment by palkan** [comment]
As I remember, the reason for using `specify` was a semantical one: when using `it`, we expect some description of the action/operation/state, since "it" is the start of a sentence; while `specify` is usually used without any description.

I think, we can add a style option to the cop; smth like:

```yml
RSpec/AggregateExamples:
  EnforcedStyle: specify # other options are: it | auto
```

Not sure how "auto" should work: what if we aggregate specs with both `it` and `specify` methods used?
